cli/src/helper.rs

```diff
@@ -27,6 +27,7 @@ pub fn get_password(prompt: &str, confirmation: bool) -> Result<Password, Error>
     password.with_prompt(prompt);
 
     if confirmation {
+        password.with_prompt("Provide a new Stronghold password");
         password.with_confirmation("Confirm password", "Password mismatch");
```

Comment:
Isn't that going to ask twice?

Reply:
no, it's like this: 
![image](https://github.com/iotaledger/iota-sdk/assets/20909273/e2aa61fb-234e-4248-aa71-d029c3dc751c)

Reply:
`with_confirmation()` should ask for the password twice, otherwise the name makes no sense

Reply:
And we are sure that confirmation ALWAYS mean new password? There are no other case where we ask for confirmation than initialisation? Otherwise the message would be weird

Reply:
I think so, I couldn't find any other case. The confirmation is done when we either change the password or create a new password to protected db.